gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -84,6 +92,12 @@ from google.iam.v1 import policy_pb2  # type: ignore
 def client_cert_source_callback():
     return b"cert bytes", b"key bytes"
 
+# TODO: use async auth anon credentials by default once the minimum version of google-auth is upgraded.
```

Comment:
Please can you file a bug to follow up on the todo comment and include the link here?

Reply:
Filed https://github.com/googleapis/gapic-generator-python/issues/2107 and added link.